index.js

```diff
@@ -34,6 +34,10 @@ module.exports = {
         "no-useless-backreference": "error",
         "require-atomic-updates": "error",
         "eqeqeq": "error",
+        "semi": ["error", "always", { "omitLastInOneLineBlock": true, }],
+        "semi-style": ["error", "last"],
+        "no-extra-semi": ["error"],
+        "semi-spacing": ["error", { "before": false, "after": true }],
```

Comment:
Why/when would we want spacing after a semicolon? I can't think of an example.

Reply:
When you have 2 expressions in the same line: `const x = 2; const y = 3;`

Reply:
I see. That makes sense.